war/HarpLogin.html

```diff
@@ -51,15 +143,98 @@
     Your web browser must have JavaScript enabled
     in order for this application to display correctly.
 </noscript>
-    <div class="login-bg-image"></div>
-    <div id="okta-login-container"></div>
+<div class="login-bg-image"></div>
+<div id="okta-login-container"></div>
+
+<form action="" method="post" id="loginForm">
+    <input type="hidden" id="loginPost" name="loginPost" value="">
+</form>
+
+<footer class="footer">
+    <div class="container">
+        <div class="row">
+            <div class="col-md-10 footer-content">
+                <div>
+                    <a title="CMS home page" tabindex="-1" href="http://www.cms.gov/" target="_blank">
+                        <h4 class="screen-reader">CMS.gov</h4>
+                        <img class="mat-logo" alt="CMS.gov" tabIndex="0" src="./images/MAT-logo_white.svg">
```

Comment:
Alt is inaccurate.

Reply:
The logo is that CMS.gov | MEASURE AUTHORING TOOL. Either way, I'll update it later. This is fine for now